### PR TITLE
 feat-558-breake-title-header-on-card-header

### PR DESCRIPTION
**Pull request by SejoB**
#558

### CODE REVIEW

File: src/utils/string.utils.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const MAX_WORDS_PER_TITLE_LINE = 5;
+
+export const splitTextHeader = (text = '') => {
+  let textLine1 = text;
```
Comment (eidan66) [inline src/utils/string.utils.ts:4]: Why text can be an empty string?
And if it can be empty, maybe return it immediately?

**Review by eidan66 — APPROVED** [review]

File: src/utils/string.utils.ts (hunk 1 above)
Comment (SejoB) [reply src/utils/string.utils.ts:4]: I put empty string here like default parameter, in this case I not need to check for undefined.
Text can come to component like undefined, in this case this function will not evaluate

File: src/components/molecules/card/CardHeader.tsx [hunk 2]
```diff
@@ -17,7 +18,7 @@ const useStyles = makeStyles({
     width: '100%',
   },
   text: {
-    width: '75%',
+    maxWidth: '75%',
```
Comment (yuvalbl) [inline src/components/molecules/card/CardHeader.tsx:21]: remove maxWidth
add spacing(3 ,2) or  spacing(2) instead

File: src/utils/string.utils.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const MAX_WORDS_PER_TITLE_LINE = 5;
+
```
Comment (yuvalbl) [inline src/utils/string.utils.ts:2]: add description plz :)

**Review by yuvalbl — APPROVED** [review]

**Pull request merged**